themes/doks/layouts/index.html

```diff
@@ -80,8 +77,7 @@ <h3 class="card-title">
                   <img src='{{ "icons/grammar-check-white.svg" | relURL }}' alt="Releases"  height="68" class="dark-only">
                 </a>
                 <span>Alerts and releases</span></h3>
-              <p>Get updates on <strong>Apache Log4j</strong> mitigation, patch releases and other release announcements.
-              <a href="en/release-notes.html" class="fw-semibold">Latest</a>
+              <p>View <a href="en/release-notes.html" class="fw-semibold">release notes</a> and associated checksum files and licence information. </p>
```

Comment:
```suggestion
              <p>View <a href="en/release-notes.html" class="fw-semibold">release notes</a> and associated checksum files and license information. </p>
```
This is a tricky one because on the licenses page, I think we use "licence". But if we're switching to American it should be s.

Reply:
good point! i just can't spell that word in any dialect...